### PR TITLE
Bug-10-Profile-Front-End-UI 

### PR DESCRIPTION
**Pull request by sergevei**

### CODE REVIEW

File: src/containers/profile/index.jsx [hunk 1]
```diff
@@ -3,24 +3,18 @@ import { Route, Switch } from 'react-router-dom';
...
-          <Profile/>
           <Switch>
-            <Route exact path="/profile" component={ProfileTab} />
-          </Switch>
-          <Switch>
-            <Route path="/profile/public" component={ProfileTab} />
-            <Route path="/profile/courses" component={CoursesTab} />
-            <Route path="/profile/account" component={AccountTab} />
-            <Route path="/profile/settings" component={SettingsTab} />
+            <Route path="/profile-public" component={ProfileTab} />
```
Comment (OlegPanasyuk) [inline src/containers/profile/index.jsx:16]: Look, You have 4 similar components. I think you need to create HOC which will get params and create the necessary component. example `<Tab type='accountTab'/>`.

File: src/containers/profile/profile.js [hunk 2]
```diff
@@ -20,29 +16,36 @@ export const Profile = () => {
...
   return (
     <>
-      <h3 className={hidden_text}>Profile's page which include courses, settings, public profile, users's account</h3>
+      <h3 className={hidden_text}>
+        Profile's page which include courses, settings, public profile, users's
+        account
+      </h3>
       <div className={mobile_burger}>
-        <button type='button' onClick={() => toggleMenu()}>
+        <button type="button" onClick={() => toggleMenu()}>
```
Comment (OlegPanasyuk) [inline src/containers/profile/profile.js]: Hm... This inner function will be created each time when you push the button. Please, move it into a class method or static function.

File: src/containers/profile/profile.js [hunk 3]
```diff
@@ -51,59 +54,55 @@ export const Profile = () => {
...
+            <Link to="/profile-public">
               <button
-                type='button'
-                className={`profile_button ${getStatus === 0? 'profile_button_active': ''}`}
-                onClick={() => {
-                  toggleButton(0), setMenu({ marginLeft: '-105%' });
-                }}>
+                type="button"
+                className={`${button} ${
+                  currentTab === tabs.public ? active_button : ''
```
Comment (OlegPanasyuk) [inline src/containers/profile/profile.js:61]: I think that calculate component class is not looks so good for understanding code.
ClassNames package was recommended as a good way to prettify our code.
Please, Make me noted about your decision about that npm-package.

https://www.npmjs.com/package/classnames

File: src/containers/profile/styles.module.scss [hunk 4]
```diff
@@ -7,6 +7,25 @@
...
   background-color: $bcg-gray-light;
 
+  * {
+    outline: none;
+  }
+
+  .button {
+    background: transparent;
+    outline: none;
+    cursor: pointer;
```
Comment (OlegPanasyuk) [inline src/containers/profile/styles.module.scss:17]: Use nested styles like as:
```
.button{
   &:hover {
           color: red;
    }
}
```

**Review by OlegPanasyuk — CHANGES_REQUESTED** [review]

File: src/containers/profile/profile.js (hunk 2 above)
Comment (sergevei) [reply src/containers/profile/profile.js]: done

**Review by slavasgod — APPROVED** [review]

File: src/containers/profile/profile.js [hunk 5]
```diff
@@ -20,29 +16,36 @@ export const Profile = () => {
...
+    settings: '/profile-settings',
+  };
 
   const toggleMenu = () => {
-    if(menus.marginLeft === '0'){
-      setMenu({ marginLeft: '-105%' })
-    }else{
-      setMenu({ marginLeft: '0' })
+    if (menus.marginLeft === '0') {
+      setMenu({ marginLeft: '-105%' });
```
Comment (OlegPanasyuk) [inline src/containers/profile/profile.js]: Why are you not using `classList.toggle('className')`?

Comment (sergevei) [reply src/containers/profile/profile.js]: done

**Review by OlegPanasyuk — APPROVED** [review]

**Pull request merged**